src/diagnostics/check-screen/check-screen.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useRef, useEffect, useMemo } from 'react'
+import { useDebouncedCallback } from '../../lib/hooks/use-debounced-callback'
+import { useBridgeSelector } from '../../helpers/context-bridge'
+import { RouteInfo } from '../../bundles/routes-types'
+
+const CheckScreen: React.FC = () => {
+  const ref = useRef<HTMLIFrameElement>(null)
+  const routeInfo = useBridgeSelector<RouteInfo>('selectRouteInfo')
+  // pass through IPFS check query params.
+  const checkIframeUrl = useMemo(() => {
+    if (routeInfo == null) {
+      return 'https://check.ipfs.network/'
+    }
+    const url = new URL(routeInfo.url.replace('/diagnostics/check', ''), 'https://check.ipfs.network/')
+
+    return url.toString()
+  }, [routeInfo])
```

Comment:
@lidel I added this as i was updating the check screen.. i see you merged that PR and added just a CID pass through. will close this PR